Admin/AbstractAdmin.php

```diff
@@ -2838,22 +2838,12 @@ public function getDashboardActions()
     }
 
     /**
-     * Setting to true will enable mosaic button for
-     * the admin screen.
-     * Setting to false will hide mosaic button for
+     * Setting to true will enable mosaic button for the admin screen. Setting to false will hide mosaic button for
      * the admin screen.
      */
-    public function setIsShowMosaicButton($isShow)
+    public function setIsShowMosaicButton($isShown)
```

Comment:
> don't want to argue with you

Why not? If you explain like you just did, you can change my mind :) So now I am not satisfied with any solution... and I think `setVisibility` would have been better